pattern_library/monkey_utils.py

```diff
@@ -11,14 +12,19 @@
 
 
 def override_tag(
-    register: django.template.Library, name: str, default_html: Optional[str] = None
+    register: django.template.Library,
+    name: str,
+    default_html: Optional[type] = UNSPECIFIED,
```

Comment:
I don't think this type annotation is right. This is saying the expected value is either `None` or a class, which means that the default value isn't valid (it's an instance). 

If we're not restricting types here (which it seems we aren't on earlier Django versions at least) then `typing.Any` would be more appropriate.